### PR TITLE
Hakeeper: Add Task Scheduler

### PR DESCRIPTION
**Pull request by w-zr**
## What type of PR is this?

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [ ] Documentation
- [x] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

Add Task Scheduler in Hakeeper

## What this PR does / why we need it:

### CODE REVIEW

File: pkg/hakeeper/task/cn_ordered_map.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+// limitations under the License.
+
+package task
+
+import (
+	"sort"
+)
+
+type OrderedMap struct {
+	Map         map[string]uint32
```
Comment (fengttt) [inline pkg/hakeeper/task/cn_ordered_map.go:22]: Do not expose these members as public.

File: pkg/hakeeper/task/cn_ordered_map.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package task
+
+import (
+	"sort"
+)
+
+type OrderedMap struct {
```
Comment (fengttt) [inline pkg/hakeeper/task/cn_ordered_map.go:21]: I would not use such a generic name for map [stirng]uint32.

File: pkg/hakeeper/task/cn_ordered_map.go (hunk 1 above)
Comment (w-zr) [reply pkg/hakeeper/task/cn_ordered_map.go]: fixed.

File: pkg/hakeeper/task/cn_ordered_map.go (hunk 2 above)
Comment (w-zr) [reply pkg/hakeeper/task/cn_ordered_map.go]: renamed

**Review by zhangxu19830126 — APPROVED** [review]

**Pull request merged**